contracts/strategies/stargate/Stargate.sol

```diff
@@ -266,4 +263,71 @@ contract Stargate is UUPSUpgradeable, StrategyOwnablePausableBaseUpgradeable {
                 .userInfo(strategyStorage.farmId, address(this))
                 .amount;
     }
+
+    function setBinStep(
+        IERC20Upgradeable tokenX,
+        IERC20Upgradeable tokenY,
+        uint256 binStep
+    ) public onlyOwner {
+        StargateStorage storage strategyStorage = StargateStorageLib
+            .getStorage();
+
+        strategyStorage.binSteps[tokenX][tokenY] = binStep;
+        strategyStorage.binSteps[tokenY][tokenX] = binStep;
```

Comment:
if you want then i guess this can be shorten into:
```
strategyStorage.binSteps[tokenX][tokenY] = strategyStorage.binSteps[tokenY][tokenX] = binStep;
```

Reply:
Thanks, but that not my type :smiling_face_with_tear: I try stick to one line, one action.